packages/manager/modules/pci/src/projects/project/storages/databases/components/input-rule/input-rule.html

```diff
@@ -0,0 +1,10 @@
+<span class="oui-password-rule">
```

Comment:
a `oui-field` or `oui-password` doesn't suit the needs ?

Reply:
`oui-password` hides the text, so it's not possible to use it here.
`oui-field` works if there is only one rule to check, but here we want to check more than one